troposphere/certificatemanager.py

```diff
@@ -1,6 +1,13 @@
 from . import AWSObject, AWSProperty
 
 
+class Tag(AWSProperty):
```

Comment:
A new Tag class should not be created for this purpose. There are actually 3 different Tag implementations already: 1) using a simple list, 2) Tags in \_\_init\_\_.py, and 3) a specialized one for autoscaling. For now you can just define it as a list which is what most are defined as in the code. Do a `grep -r Tags troposphere` to see the various implementations.